tests/cypress/integration/actions_users/issue_1810_login_logout.js

```diff
@@ -72,25 +85,7 @@ context('When clicking on the Logout button, get the user session closed.', () =
                 cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`);
                 cy.contains('.cvat-task-details-task-name', `${taskName}`).should('be.visible');
             });
-        });
-
-        it('Incorrect user and correct password', () => {
             cy.logout();
-            login('randomUser123', Cypress.env('password'));
-            cy.url().should('include', '/auth/login');
-            cy.closeNotification('.cvat-notification-notice-login-failed');
-        });
-
-        it('Correct user and incorrect password', () => {
-            login(Cypress.env('user'), 'randomPassword123');
-            cy.url().should('include', '/auth/login');
-            cy.closeNotification('.cvat-notification-notice-login-failed');
-        });
-
-        it('Incorrect user and incorrect password', () => {
-            login('randomUser123', 'randomPassword123');
-            cy.url().should('include', '/auth/login');
-            cy.closeNotification('.cvat-notification-notice-login-failed');
```

Comment:
Here should be invalid credentials check. I assume in our case it should be existing user and not existing wallet address or vice versa